src/common/actions/repositories.js

```diff
@@ -42,44 +39,27 @@ export function fetchUserRepositories(pageNumber) {
           if (result.status !== 'success') {
             throw getError(response, result);
           }
-          dispatch(setRepositories(result.payload.repos));
-          dispatch(fetchRepositoriesSuccess());
-
-          if (result.pageLinks) {
-            dispatch(setPageLinks(result.pageLinks));
```

Comment:
I think this should happen in the reducer, I've done it very simply for now but a refactor would move all of this to the reducer.